WcaOnRails/app/models/competition.rb

```diff
@@ -476,9 +477,6 @@ def info_for(user)
     if !self.results_posted? && self.is_probably_over? && !self.cancelled?
       info[:upload_results] = I18n.t('competitions.messages.upload_results')
     end
-    if self.in_progress? && !self.cancelled?
```

Comment:
Why are you entirely deleting this, instead of nesting another `if` query that decides _which_ info message to display?

See also my comment on `competitions/_nav.html.erb` below.

Reply:
Because I wanted to add variable inside the string, which is possible in erb, but I couldn't find any way to do it in rb file. 😬 Is there any way for that?

Reply:
I haven't tried it out, but from the top of my head the following could work:

```ruby
if self.in_progress? && !self.cancelled?
  if self.is_using_wca_live_for_scoretaking?
    # your new message with the link
    wca_live_link = link_to(t('common.here'), self.get_wca_live_link)
    info[:in_progress] = t('competitions.messages.in_progress_at_wca_live_html', link_here: wca_live_link)
  else
    # stuff that already was there before your change
    info[:in_progress] = I18n.t('competitions.messages.in_progress', date: I18n.l(self.end_date, format: :long))
  end
end
```

Reply:
If the `link_to` part is the actual problem, then you might want to embed the `<a href>` part (that link_to creates for you as a convenience method) directly into the raw i18n string in the YML. We do this in several other places, simply CTRL+F search for `href` anywhere in `en.yml`.